.gitignore

```diff
@@ -27,3 +27,10 @@ override.tf.json
 
 # Include tfplan files to ignore the plan output of command: terraform plan -out=tfplan
 # example: *tfplan*
+
+# Ignore pre-commit hook configs
+.pre-commit-config.yaml
```

Comment:
why we ignore it?

Reply:
will remove. I initially thought it would just be needed for local config.